src/styles/getReportActionContextMenuStyles.js

```diff
@@ -25,10 +25,13 @@ const bigWrapperStyle = [
  * Generate the wrapper styles for the ReportActionContextMenu.
  *
  * @param {Boolean} isMini
+ * @param {Boolean} isSmallScreenWidth
  * @returns {Array}
  */
-function getReportActionContextMenuStyles(isMini) {
-    return isMini ? miniWrapperStyle : bigWrapperStyle;
+function getReportActionContextMenuStyles(isMini, isSmallScreenWidth) {
+    // For the bottom_docked modal type, we have defined default paddings in getBaseModalStyles file, so there is no need to add vertical padding for small screen width
+    // When popover modal type, then only Need to pass vertical padding.
+    return isMini ? miniWrapperStyle : [...bigWrapperStyle, isSmallScreenWidth ? {} : styles.pv3];
```

Comment:
```suggestion
    if (isMini) {
        return miniWrapperStyle;
    }

    // Small screens use a bottom-docked modal that already has vertical padding.
    return [...bigWrapperStyle, isSmallScreenWidth ? {} : styles.pv3];
```

I think this is better for readability

Reply:
Updated